autoagora/main.py

```diff
@@ -35,6 +36,21 @@ async def allocated_subgraph_watcher():
         (args.relative_query_costs_exclude_subgraphs or "").split(",")
     )
 
+    try:
+        pgpool = await asyncpg.create_pool(
+            host=args.postgres_host,
```

Comment:
from my previous experience, pool size should be controlled (from args ?)
https://magicstack.github.io/asyncpg/current/api/index.html#asyncpg-api-pool

Reply:
Yes, you're right, I should make this a user setting.
In general, what do you think of limiting to a single connection by default? None of the code paths are extremely time-sensitive, and I don't expect cyclical dependencies.

Reply:
https://github.com/semiotic-ai/autoagora/pull/54/commits/096087fbc4ec01a763737d6b4c2714a6da12d816